apps/forge/src/routes/_authenticated/_reponly/users/$id.tsx

```diff
@@ -15,7 +14,7 @@ import { Check, X } from "lucide-react";
 export default function Component() {
   const data = Route.useLoaderData();
   const { user, trainings, signIns } = data!;
-  const rep = useUserRoles().includes("Rep");
+  const rep = user!.roles.some((role) => role.name === "Rep");
```

Comment:
Was this intentional?

Reply:
i goofed a lil while back which was when the tables were broken a couple days ago, was being overzealous with the useUserRoles refactor lmao

Reply:
https://discord.com/channels/570657514572742666/1007356986041569340/1308778940030128138